runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -623,12 +623,12 @@ MM_IncrementalGenerationalGC::heapRemoveRange(MM_EnvironmentBase *env, MM_Memory
  * @see MM_GlobalCollector::heapReconfigured()
  */
 void
-MM_IncrementalGenerationalGC::heapReconfigured(MM_EnvironmentBase *env)
+MM_IncrementalGenerationalGC::heapReconfigured(MM_EnvironmentBase *env, HeapReconfigReason reason, MM_MemorySubSpace *subspace, void *lowAddress, void *highAddress)
 {
 	MM_EnvironmentVLHGC *envVLHGC = MM_EnvironmentVLHGC::getEnvironment(env);
 	
-	_reclaimDelegate.heapReconfigured(envVLHGC);
-	_schedulingDelegate.heapReconfigured(envVLHGC);
+	_reclaimDelegate.heapReconfigured(envVLHGC, reason, subspace, lowAddress, highAddress);
+	_schedulingDelegate.heapReconfigured(envVLHGC, reason, subspace, lowAddress, highAddress);
```

Comment:
Don't pass it here, since they don't use it. Hence don't touch the delegate classes.